core/src/main/java/edu/bsu/storygame/core/view/NotebookLayer.java

```diff
@@ -298,6 +300,48 @@ public void onEmit(Button button) {
         }
     }
 
+    private final class SkillGroup extends Group {
+
+        private SkillGroup() {
+            super(AxisLayout.horizontal().offStretch());
+            updatePlayerSkills(player);
+            player.skills.connect(new RList.Listener<Skill>() {
+                @Override
+                public void onAdd(Skill skill) {
+                    SkillGroup.this.updatePlayerSkills(player);
+                }
+
+                @Override
+                public void onRemove(Skill skill) {
+                    SkillGroup.this.updatePlayerSkills(player);
+                }
+            });
+        }
+
+        private void updatePlayerSkills(Player player) {
```

Comment:
You don't need to pass in the player here since there is a reference to it at the top of the file.